tests/util.py

```diff
@@ -0,0 +1,20 @@
+# This method will be used by the mock to replace requests.get
+def mocked_requests_get_wrapper(response_dict):
```

Comment:
I like having a generic requests_get_mock creator so that it can be used more diversely in the future.